### DISCUSSION TITLE
Volumes and chapters

### DISCUSSION THREAD

**Discussion by bookimp** [lncrawl/lightnovel-crawler#2210] [Q&A]
Is there a way (an easy way I don't speak phython so if it's complciated just tell me) to have it not group the chapters into volumes?

**Comment by thibaud200** [comment]
the crawler does not propose to download chapter by chapter or to change de number of chapter per volume witch is 100 by defaut or one big file with everything. there was a similar ask for it. and i did a small script to do both(its not necessarily optimize but it should do the work : go see https://github.com/dipu-bd/lightnovel-crawler/discussions/2365
